index.bs

```diff
@@ -2966,6 +2966,16 @@ between [=navigables=] and device pixel ratio overrides. It is initially empty.
 Note: this map is not cleared when the final session ends i.e. device pixel
 ratio overrides outlive any WebDriver session.
 
+A <dfn for="viewport-configuration">viewport dimensions</dfn> is a [=struct=] with an [=struct/item=] named
+<dfn attribute for="viewport-dimensions">height</dfn> which is an integer and
+a [=struct/item=] named <dfn attribute for="viewport-dimensions">width</dfn> which is an integer.
+
+A <dfn>viewport configuration</dfn> is a [=struct=] with an [=struct/item=] named
```

Comment:
So I am still not sure if a struct is suitable here and I am not sure if we can check if a field is present in a struct. 

https://infra.spec.whatwg.org/#structs says that each item needs to be of a defined type which I interpret as it cannot be null and needs to have a value of a given type. WDYT? cc @jgraham

Reply:
Nullable values are definitely allowed in structs (HTML uses them, for example). I think infra is just saying you're supposed to specify what the type is when you define the struct, but the equivalent of `Option<T>` (or `T | null` if that's your preferred syntax) is fine :)

Reply:
So to check if there is a value we should assume the struct item is always there and check if it is null?  That SGTM.